src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageVersionsModel.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI.Models.Package
+{
+    public class PackageVersionsModel
+    {
+        private INuGetSearchService _nuGetSearchService;
+        private IReadOnlyCollection<VersionInfoContextInfo>? _availableVersions;
+        private PackageModel _primaryPackage;
```

Comment:
What does a primary mean different from a base package?

```suggestion
        private PackageModel _basePackage;
```

Reply:
Hmm I've been struggling with what to name this, so I removed it all together.